pkg/config/parameters.go

```diff
@@ -423,6 +423,15 @@ type NetworkParameters struct {
 	XffNumTrustedHops uint32 `yaml:"num-trusted-hops"`
 }
 
+// ListenerParameters hold various configurable listener values.
+type ListenerParameters struct {
+	// ConnectionBalancer. If the value is exact, the listener will use the exact connection balancer
+
+	// See https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/listener.proto#envoy-api-msg-listener-connectionbalanceconfig
+	// for more information.
+	ConnectionBalancer string `yaml:"connection-balancer"`
```

Comment:
Nit: I generally prefer k8s-style `camelCase` YAML names, but I see we have plenty of hyphenated ones in our config filie. So this is a small nit, I'll leave it up to other maintainers to make a call.